wagtail/admin/ui/side_panels.py

```diff
@@ -155,19 +160,30 @@ def get_lock_context(self):
             "locking_enabled": self.locking_enabled,
         }
 
+    def get_usage_context(self):
+        return {
+            "usage_count": ReferenceIndex.get_references_to(self.object)
+            .group_by_source_object()
+            .count(),
+            "usage_url": getattr(self.object, "usage_url", None),
```

Comment:
Need to use `getattr` because I didn't want to add a `usage_url` property to `Page` to avoid making it bigger, considering we can just do `reverse("wagtailadmin_pages:usage", args=(page.id,))` like we do for other urls in `PageStatusSidePanel.get_context_data()`